### PR TITLE
nearestBuilding --> nearestObjects

### PR DESCRIPTION
**Pull request by dardothemaster**
Nearestobjects allows detecting of editor placed houses, and it's around 200 times faster than nearestBuilding.

### CODE REVIEW

**Comment by NiiRoZz** [comment]
You have already tested this ?

**Comment by tkcjesse** [comment]
Also, I would suggest redoing this as a singular commit.

File: life_hc/MySQL/Housing/fn_fetchPlayerHouses.sqf [hunk 1]
```diff
@@ -22,7 +22,7 @@ _containers = [_query,2,true] call HC_fnc_asyncCall;
 _containerss = [];
 {
 	_position = call compile format["%1",_x select 1];
-	_house = nearestBuilding _position;
+	_house = nearestObjects [_pos,["House"], 16] select 0;
```
Comment (NiiRoZz) [inline life_hc/MySQL/Housing/fn_fetchPlayerHouses.sqf]: Men you don't have tested look this

File: life_server/Functions/Housing/fn_fetchPlayerHouses.sqf [hunk 2]
```diff
@@ -20,7 +20,7 @@ _containers = [_query,2,true] call DB_fnc_asyncCall;
 _containerss = [];
 {
 	_position = call compile format["%1",_x select 1];
-	_house = nearestBuilding _position;
+	_house = nearestObjects [_pos,["House"], 16] select 0;
```
Comment (NiiRoZz) [inline life_server/Functions/Housing/fn_fetchPlayerHouses.sqf]: Too here

**Pull request closed without merging**